config/client.go

```diff
@@ -310,7 +310,13 @@ func GenerateLimitedClientConfig(c *model.Config, telemetryID string, license *m
 	props["GuestAccountsEnforceMultifactorAuthentication"] = strconv.FormatBool(*c.GuestAccountsSettings.EnforceMultifactorAuthentication)
 
 	props["EnableSignUpWithLTI"] = strconv.FormatBool(false)
+	//
+	//	Check that if LTI Plugin is installed.
+	//
 	if LTIConfig, ok := c.PluginSettings.Plugins[model.LTI_PLUGIN_ID]; ok {
+		//
+		//	Check that if LTI Authentication is enabled.
+		//
```

Comment:
You don't have to add comments about things that can be easily understood from readable code.

Instead, explain why these changes are needed.

E.g. here, you could mention what this code block does, i.e. add a client config parameter to allow the webapp to check if signup with LTI is enabled.